### PR TITLE
fix: layout issue fixed for small screen sizes

### PR DESCRIPTION
**Pull request by osamaasifoak**
[Scrolling issue for small screens#486](https://github.com/meditohq/medito-app/issues/486)

### CODE REVIEW

File: lib/views/player/widgets/overlay_cover_image_widget.dart [hunk 1]
```diff
@@ -9,8 +9,8 @@ class OverlayCoverImageWidget extends StatelessWidget {
     var size = MediaQuery.of(context).size;
 
     return Container(
-      height: size.width,
-      padding: EdgeInsets.symmetric(horizontal: 15, vertical: 15),
+      height: size.width - 32,
```
Comment (michaelcspeed) [inline lib/views/player/widgets/overlay_cover_image_widget.dart]: Magic numbers!

File: android/app/build.gradle [hunk 2]
```diff
@@ -108,6 +108,15 @@ android {
             signingConfig signingConfigs.release
         }
     }
+    configurations.all {
+    resolutionStrategy {
+        eachDependency {
+            if ((requested.group == "org.jetbrains.kotlin") && (requested.name.startsWith("kotlin-stdlib"))) {
+                useVersion("1.8.0")
+            }
+        }
```
Comment (michaelcspeed) [inline android/app/build.gradle:117]: What's all this for?

Comment (osamaasifoak) [reply android/app/build.gradle:117]: I was facing an issue while running the app. I'm attaching a link for a better explanation.

1. https://github.com/flutter/flutter/issues/119247
2. https://developer.android.com/build/dependencies#resolution_errors

Comment (michaelcspeed) [reply android/app/build.gradle:117]: Ok thanks for the info!

**Review by michaelcspeed — APPROVED** [review]

**Pull request merged**